### PR TITLE
Difference

### PR DESCRIPTION
**Pull request by guru-ricoh**

### CODE REVIEW

File: main.java [hunk 1]
```diff
@@ -0,0 +1,3 @@
+public void main(String args[]){
+System.out.println("This is main programme");
```
Comment (LoicB) [inline main.java:2]: This is the way to print. Well done! Your future at Ricoh is ready!

**Review by LoicB — COMMENTED** [review]
// this is a comment

File: main.java (hunk 1 above)
Comment (guru-ricoh) [reply main.java:2]: Thank you for the review. Actually it was very difficult code,

Comment (guru-ricoh) [reply main.java:2]: :-)

Comment (LoicB) [reply main.java:2]: 🥇

**Pull request merged**